examples/cms-agilitycms/components/agility-pageModules/FeaturedPost.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Link from "next/link";
+import truncate from "truncate-html";
+import { AgilityImage } from "@agility/nextjs"
+
+const FeaturedPost = ({ module }) => {
+  // get module fields
+  const { fields } = module;
+
+  // get featured post
+  const { featuredPost } = fields;
+
+  // convert date
+  const dateStr = new Date(featuredPost?.fields.date).toLocaleDateString();
+
+  // truncate post content
+  const description = truncate(featuredPost?.fields.content, {
+    length: 160,
+    decodeEntities: true,
+    stripTags: true,
+    reserveLastWord: true,
+  });
+
+
+  // return null if no featured post is selected
+  if (!featuredPost) {
+    return null;
+  }
+
+  return (
+    <div className="relative px-8 mb-8">
+      <div className="flex flex-col sm:flex-row max-w-screen-xl mx-auto pt-8 group">
+        <div className="sm:w-1/2 lg:w-2/3 sm:rounded-t-none sm:rounded-l-lg relative">
+          <Link href={`/blog/${featuredPost.fields.slug}`}>
+            <a className="cursor-pointer">
+              <div className="h-64 sm:h-96 relative">
+                <AgilityImage
```

Comment:
Any insight why Agility image over next/image?

Reply:
@leerob [AgilityImage](https://github.com/agility/agility-next/blob/main/src/AgilityImage.tsx) actually uses next/image behind the scenes!  It avoids the need for any other config as it just changes the image URL parameters to resize it or change format and quality on our CDN